pkg/ibmvpc/examples/out/explain_out/QueryConnectionSGBasic3_all_vpcs_explain_debug.txt

```diff
@@ -1,5 +1,5 @@
-Explaining connectivity from crn:v1:staging:public:is:us-south:a/6527::vpc:a456 to 161.26.0.0/20 within test-vpc1-ky using "protocol: UDP src-ports: 10-100 dst-ports: 443"
-===========================================================================================================================================================================
+Explaining connectivity from vsi1-ky to 161.26.0.0/20 within test-vpc1-ky using "protocol: UDP src-ports: 10-100 dst-ports: 443"
```

Comment:
Same comment. src input here is `crn:v1:staging:public:is:us-south:a/6527::vpc:a456` and dst is `161.26.0.0/20`. This should be part of the header